docs/cookbook/dump-variables.md

```diff
@@ -0,0 +1,16 @@
+# Dump variables in framework
+Shopsys framework uses symfony with its VarDumper. In addition to the standard function `dump()`, we implement the function `d()`, 
+which you can pass any number of attributes where each one is dumped.
+
+Dumped variables are primarily available in the profiler. 
+
+Since we don't want to dump variables into the json response when developing the frontend api, the dumps are directed to the symfony dump server. (https://symfony.com/doc/current/components/var_dumper.html#the-dump-server)
```

Comment:
```suggestion
Since we do not want to dump variables into the JSON response when developing the frontend API, the dumps are directed to the Symfony dump server. (https://symfony.com/doc/current/components/var_dumper.html#the-dump-server)
```

Reply:
It is more formal way to write do not vs don't in English.

https://allthedifferences.com/difference-between-do-not-and-dont/

Reply:
ok